app/src/main/java/travelbook/android/app/data/TravelBookDatabaseOpenHelper.java

```diff
@@ -0,0 +1,69 @@
+package travelbook.android.app.data;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+/**
+ * A data class for working with the database.
+ */
+public class TravelBookDatabaseOpenHelper extends SQLiteOpenHelper {
+
+    public static final class MemoriesContract {
+        private MemoriesContract() {
+        }
+
+        public static class MemoriesEntry {
+            public static final String TABLE_NAME = "travelbook.db";
+            public static final String COLUMN_NAME_MEMORY_ID = "memory_id";
+            public static final String COLUMN_NAME_MEMORY_DESC = "memory_desc";
+        }
+    }
+
+    public static final class PhotosContract {
+        private PhotosContract() {
+        }
+
+        public static class PhotosEntry {
+            public static final String TABLE_NAME = "Photos";
+            public static final String COLUMN_NAME_PHOTO_ID = "photo_id";
+            public static final String COLUMN_NAME_PHOTO_URI = "photo_uri";
+            public static final String COLUMN_NAME_PHOTO_DESC = "photo_desc";
+            public static final String COLUMN_NAME_IS_PRIMARY = "is_primary";
+            public static final String COLUMN_NAME_MEMORY = "memory";
+        }
+    }
+
+    private static final String SQL_CREATE_MEMORY_ENTRIES =
```

Comment:
Move SQL_CREATE_MEMORY_ENTRIES into MemoriesContract and rename to SQL_CREATE_ENTRIES